.ci/jenkins/Jenkinsfile.deploy

```diff
@@ -139,13 +139,29 @@ pipeline {
                         } else {
                             installOrDeploy = 'install'
                         }
+
+                        def mavenCommand = getOptaplannerMavenCommand()
+                           .withProperty('maven.test.failure.ignore', true)
+                           .withProperty('operator.image.build')
+                           .skipTests(params.SKIP_TESTS)
+                           .withSettingsXmlFile(MAVEN_SETTINGS_FILE)
```

Comment:
needs to be accessed within configFileProvider block.

Reply:
Fixed.